modules/drivers/databricks-jdbc/test/metabase/driver/databricks_jdbc_test.clj

```diff
@@ -0,0 +1,77 @@
+(ns metabase.driver.databricks-jdbc-test
+  (:require
+   [clojure.test :refer :all]
+   [java-time.api :as t]
+   [metabase.driver :as driver]
+   [metabase.driver.databricks-jdbc :as databricks-jdbc]
+   [metabase.driver.sql-jdbc.connection :as sql-jdbc.conn]
+   [metabase.test :as mt]
+   [toucan2.core :as t2]))
+
+(deftest ^:parallel sync-test
+  (testing "`driver/describe-database` implementation returns expected resutls."
+    (mt/test-driver
+      :databricks-jdbc
+      (is (= {:tables
+              #{{:name "venues", :schema "test-data", :description nil}
+                {:name "checkins", :schema "test-data", :description nil}
+                {:name "users", :schema "test-data", :description nil}
+                {:name "people", :schema "test-data", :description nil}
+                {:name "categories", :schema "test-data", :description nil}
+                {:name "reviews", :schema "test-data", :description nil}
+                {:name "orders", :schema "test-data", :description nil}
+                {:name "products", :schema "test-data", :description nil}}}
+             (driver/describe-database :databricks-jdbc (mt/db)))))))
+
+(mt/defdataset dataset-with-ntz
+  [["table_with_ntz" [{:field-name "timestamp"
+                       :base-type {:native "timestamp_ntz"}}]
+    [[(t/local-date-time 2024 10 20 10 20 30)]]]])
+
+(deftest timestamp-ntz-ignored-test
+  (mt/test-driver
+   :databricks-jdbc
+   (mt/dataset
+    dataset-with-ntz
+    (testing "timestamp column was ignored during sync"
+      (let [columns (t2/select :model/Field :table_id (t2/select-one-fn :id :model/Table :db_id (mt/id)))]
+        (is (= 1 (count columns)))
+        (is (= "id" (:name (first columns)))))))))
+
+(deftest ^:parallel db-default-timezone-test
+  (mt/test-driver
+   :databricks-jdbc
+   (testing "`test-data` timezone is `Etc/UTC`"
+     (is (= "Etc/UTC" (:timezone (mt/db)))))))
+
+(deftest ^:synchronized date-time->results-local-date-time-test
+  (mt/test-driver
+    :databricks-jdbc
+    (mt/with-metadata-provider (mt/id)
+      (mt/with-results-timezone-id "America/Los_Angeles"
+        (let [expected (t/local-date-time 2024 8 29 10 20 30)]
+          (testing "LocalDateTime is not modified"
```

Comment:
I would prefer to see this tested with a query through the QP.

Reply:
I've added the `metabase.driver.databricks-jdbc-test/timezone-in-set-and-read-functions-test`.